proposals/2021-10-community_pool-minimum_deposit/README.md

```diff
@@ -0,0 +1,25 @@
+### Abstract
+
+We would like to change two parameters as follows.
+- Change the community tax from 0% to 2%.
+- Change the minimum proposal deposit from 100,000 MED to 10,000 MED.
+
+
+### Background
+
+Community pool is a means of funding participants who contribute to the Panacea ecosystem.<br/>
+As other Cosmos-based chains, we would like to use the community pool through governance votings to support contributions which boost the Panacea ecosystem, such as outstanding improvements, events and marketing.<br/>
+You can find various use-cases of the community pool from other Cosmos-based chains: https://github.com/medibloc/panacea-core/issues/186#issuecomment-927537658. 
+
+Although staking rewards of validators and delegators will be slightly reduced since the 2% of transactions fees will go to the community pool, the raised funds in the community pool will surely accelerate the Panacea ecosystem.
```

Comment:
```suggestion
Staking rewards of validators and delegators will be slightly reduced since the 2% of staking rewards (= block rewards + tx fees) will go to the community pool. The raised funds in the community pool will surely accelerate the Panacea ecosystem though.
```
The community tax affects the reward distribution by the `AllocateTokens()`:
https://github.com/cosmos/cosmos-sdk/blob/33e7297c791980428d0c698989ffa760284258f2/x/distribution/keeper/allocation.go#L81

It was confusing that the community tax ratio is applied to the `feeCollected` value. But, I also found that the block provision is also added to the `feeCollected`: 
https://github.com/cosmos/cosmos-sdk/blob/dda9bd8333774d70d330f43fa7f599e85f6c26aa/x/mint/abci.go#L37